### ISSUE TITLE
wiki is out of date (I think)

### ISSUE THREAD

**Issue by bab-cat-OWO** [tvanprooyen/mbm#34]
there are serval thing not on the wiki like wet sand or big sand stone or the crafting recipe for petrified log, and there are serval thing that just don't exist like mini block. also a lot of pages are empty.

**Comment by bab-cat-OWO** [comment]
btw I'm not angry

**Comment by tvanprooyen** [comment]
Yeah, the wiki is quite a work in progress. I haven't updated it in a long time. I just started to resume filling things in.
